stdlib/jdbc/src/test/java/org/ballerinax/jdbc/actions/CallTest.java

```diff
@@ -52,7 +52,7 @@ public void setup() {
         result = BCompileUtil.compile(Paths.get("test-src", "actions", "call_test.bal").toString());
     }
 
-    @Test(groups = CALL_TEST)
+    @Test(groups = CALL_TEST, enabled = false)
```

Comment:
Why this test is disabled?

Reply:
This test was failed with the following error
```
ballerina-jdbc-test-suite > ballerina-jdbc-test > org.ballerinax.jdbc.actions.CallTest > testCallWithStringTypes FAILED
    org.ballerinalang.util.exceptions.BLangRuntimeException: error: {ballerina/lang.array}InherentTypeViolation message=incompatible types: expected 'table<$anonType$0>[]|()|error', found 'handle'
        at call_test:testCallWithStringTypes(call_test.bal:72)
        at org.ballerinalang.test.util.BRunUtil.invoke(BRunUtil.java:410)
        at org.ballerinalang.test.util.BRunUtil.invokeFunction(BRunUtil.java:1122)
        at org.ballerinalang.test.util.BRunUtil.invoke(BRunUtil.java:163)
        at org.ballerinax.jdbc.actions.CallTest.testCallWithStringTypes(CallTest.java:57)

        Caused by:
        org.ballerinalang.util.exceptions.BLangRuntimeException: error: {ballerina/lang.array}InherentTypeViolation message=incompatible types: expected 'table<$anonType$0>[]|()|error', found 'handle'
                at call_test:testCallWithStringTypes(call_test.bal:72)
```
Now this issue is fixed with this PR https://github.com/ballerina-platform/ballerina-lang/pull/20097, I'll enable this test.